dbt_subprojects/dex/models/_projects/zeroex/arbitrum/zeroex_arbitrum_settler_addresses.sql

```diff
@@ -2,9 +2,15 @@
     config(
         schema = 'zeroex',
         alias = 'arbitrum_settler_addresses',
-        materialized = 'table'
+        materialized='incremental',
+        partition_by = ['begin_block_time'],
```

Comment:
```suggestion
```

Reply:
these tables are small, lets avoid partitions. for example, arbitrum is only 20 rows